tests/fixtures/require-space-after-keywords/good/example.js

```diff
@@ -21,12 +21,12 @@ switch (x) {
 
 try {
   x();
-} catch(error) {
+} catch (error) {
```

Comment:
Sorry for the confusion. I had retracted my comment about `catch` [here](https://github.com/DockYard/eslint-plugin-ember-suave/pull/11#issuecomment-235670286). There's an [override](http://eslint.org/docs/rules/keyword-spacing#overrides) option we can use to disallow spaces after `catch`.